completions/fish/xsubpkg.fish

```diff
@@ -0,0 +1,8 @@
+function __xsubpkg_print_packages
+    path basename -- (xdistdir)/srcpkgs/* | string replace -r '$' '\tPackage'
+end
```

Comment:
This better be put in a separate file to be used by others completions, because quite a bunch will need it.